lib/config/recommended.js

```diff
@@ -12,6 +12,8 @@ export default {
     'no-args-paths': 'error',
     'no-arguments-for-html-elements': 'error',
     'no-aria-hidden-body': 'error',
+    // Error on next major
+    'no-at-ember-render-modifiers': 'warn',
```

Comment:
We can add this as a `recommended` rule in the next major release but we don't typically `warn` for any rules so I would avoid adding it to this file.